### PR TITLE
23229 unit tests

### PR DESCRIPTION
**Pull request by cohnjesse**
## Description
[Original Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/23229)

In order to make sure our form works as intended we need to unit test the individual form pages using unit tests. This PR adds those unit tests for the Certificate of Eligibility form. I purposely left the document upload page untested as there will be massive changes in how that page works coming in the near future and it would be better to wait for those changes to be figured out before we set up unit tests for that page.

## Testing done
Unit tests added

## Screenshots

Screen Shot 2021-07-19 at 10 22 28 AM

## Acceptance criteria
- [x]  tests are written and passing
- [x]  PR submitted into vets-website

### CODE REVIEW

File: src/applications/lgy/coe/tests/config/decoration.unit.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+import {
+  DefinitionTester,
+  getFormDOM,
+} from 'platform/testing/unit/schemaform-utils.jsx';
+import createCommonStore from 'platform/startup/store';
+import formConfig from '../../config/form.js';
+
+const defaultStore = createCommonStore();
+
+describe('COE applicant communication preferences', () => {
```
Comment (micahchiang) [inline src/applications/lgy/coe/tests/config/decoration.unit.spec.jsx:15]: This should be updated to applicant service decoration

File: src/applications/lgy/coe/tests/config/documentScreener.unit.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+import {
+  DefinitionTester,
+  getFormDOM,
+} from 'platform/testing/unit/schemaform-utils.jsx';
+import createCommonStore from 'platform/startup/store';
+import formConfig from '../../config/form.js';
+
+const defaultStore = createCommonStore();
+
+describe('COE applicant communication preferences', () => {
```
Comment (micahchiang) [inline src/applications/lgy/coe/tests/config/documentScreener.unit.spec.jsx:15]: This should be updated to document screener

File: src/applications/lgy/coe/tests/config/existingLoanScreener.unit.spec.jsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+import {
+  DefinitionTester,
+  getFormDOM,
+} from 'platform/testing/unit/schemaform-utils.jsx';
+import createCommonStore from 'platform/startup/store';
+import formConfig from '../../config/form.js';
+
+const defaultStore = createCommonStore();
+
+describe('COE applicant communication preferences', () => {
```
Comment (micahchiang) [inline src/applications/lgy/coe/tests/config/existingLoanScreener.unit.spec.jsx:15]: This should be updated to loan screener

File: src/applications/lgy/coe/tests/config/history.unit.spec.jsx [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+import {
+  DefinitionTester,
+  getFormDOM,
+} from 'platform/testing/unit/schemaform-utils.jsx';
+import createCommonStore from 'platform/startup/store';
+import formConfig from '../../config/form.js';
+
+const defaultStore = createCommonStore();
+
+describe('COE applicant communication preferences', () => {
```
Comment (micahchiang) [inline src/applications/lgy/coe/tests/config/history.unit.spec.jsx:15]: This should be updated to service history

File: src/applications/lgy/coe/tests/config/loanHistory.unit.spec.jsx [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+import {
+  DefinitionTester,
+  getFormDOM,
+} from 'platform/testing/unit/schemaform-utils.jsx';
+import createCommonStore from 'platform/startup/store';
+import formConfig from '../../config/form.js';
+
+const defaultStore = createCommonStore();
+
+describe('COE applicant communication preferences', () => {
```
Comment (micahchiang) [inline src/applications/lgy/coe/tests/config/loanHistory.unit.spec.jsx:15]: This should be updated to loan history

File: src/applications/lgy/coe/tests/config/loanIntent.unit.spec.jsx [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+import {
+  DefinitionTester,
+  getFormDOM,
+} from 'platform/testing/unit/schemaform-utils.jsx';
+import createCommonStore from 'platform/startup/store';
+import formConfig from '../../config/form.js';
+
+const defaultStore = createCommonStore();
+
+describe('COE applicant communication preferences', () => {
```
Comment (micahchiang) [inline src/applications/lgy/coe/tests/config/loanIntent.unit.spec.jsx:15]: This should be updated to loan intent

File: src/applications/lgy/coe/tests/config/status.unit.spec.jsx [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+import {
+  DefinitionTester,
+  getFormDOM,
+} from 'platform/testing/unit/schemaform-utils.jsx';
+import createCommonStore from 'platform/startup/store';
+import formConfig from '../../config/form.js';
+
+const defaultStore = createCommonStore();
+
+describe('COE applicant communication preferences', () => {
```
Comment (micahchiang) [inline src/applications/lgy/coe/tests/config/status.unit.spec.jsx:15]: This should be updated to service status

**Review by micahchiang — APPROVED** [review]
lgtm just be sure to update the describe block descriptions before merging.

File: src/applications/lgy/coe/tests/config/decoration.unit.spec.jsx (hunk 1 above)
Comment (cohnjesse) [reply src/applications/lgy/coe/tests/config/decoration.unit.spec.jsx:15]: updated

File: src/applications/lgy/coe/tests/config/documentScreener.unit.spec.jsx (hunk 2 above)
Comment (cohnjesse) [reply src/applications/lgy/coe/tests/config/documentScreener.unit.spec.jsx]: updated

File: src/applications/lgy/coe/tests/config/existingLoanScreener.unit.spec.jsx (hunk 3 above)
Comment (cohnjesse) [reply src/applications/lgy/coe/tests/config/existingLoanScreener.unit.spec.jsx]: updated

File: src/applications/lgy/coe/tests/config/history.unit.spec.jsx (hunk 4 above)
Comment (cohnjesse) [reply src/applications/lgy/coe/tests/config/history.unit.spec.jsx]: updated

File: src/applications/lgy/coe/tests/config/loanHistory.unit.spec.jsx (hunk 5 above)
Comment (cohnjesse) [reply src/applications/lgy/coe/tests/config/loanHistory.unit.spec.jsx]: updated

File: src/applications/lgy/coe/tests/config/loanIntent.unit.spec.jsx (hunk 6 above)
Comment (cohnjesse) [reply src/applications/lgy/coe/tests/config/loanIntent.unit.spec.jsx]: updated

File: src/applications/lgy/coe/tests/config/status.unit.spec.jsx (hunk 7 above)
Comment (cohnjesse) [reply src/applications/lgy/coe/tests/config/status.unit.spec.jsx]: updated

**Pull request merged**